app/controllers/application_controller/timelines.rb

```diff
@@ -208,7 +205,8 @@ def tl_build_timeline_report_options
       params = params.concat(params2)
       @report.where_clause = [cond, *params]
       @report.rpt_options ||= {}
-      @report.rpt_options[:categories] = @tl_options.mngt.categories
+      @report.rpt_options[:categories] = @tl_options.management_events? ?
+        @tl_options.mngt.categories : @tl_options.policy.categories
```

Comment:
Can we please not use abbreviations for the tl_options?  `mngt` is so not intuitive as the abbreviation for `management`